engine/src/flutter/lib/ui/semantics.dart

```diff
@@ -1575,6 +1596,9 @@ class SemanticsFlags extends NativeFieldWrapperClass1 {
   /// Checks if any of the boolean semantic flags are set to true
   /// in both this instance and the [other] instance.
   bool hasRepeatedFlags(SemanticsFlags other) {
+    // isAccessibilityFocusBlocked is not checked in `hasRepeatedFlags` because
+    // two semantics nodes both with `isAccessibilityFocusBlocked` set to true is
+    // still compatible and not conflicting.
```

Comment:
In this case, the method name no longer correct. Consider rename to something more descriptive

Reply:
we should create a new method and deprecate this one.